app/views/additional_details/_document_management_submissions.html.haml

```diff
@@ -15,30 +15,33 @@
-                %span.glyphicon.glyphicon-modal-window
-        %td
-          .text-center
-            = service.identity_submission(current_identity).present? ? "True" : "False"
-        %td
-          .text-center
-            - if service.identity_submission(current_identity).present?
-              = link_to edit_service_additional_details_submission_path(service, service.identity_submission(current_identity).first, sr_id: @service_request.id), remote: true, class: 'btn btn-warning' do
-                %span.glyphicon.glyphicon-edit
-        %td
-          .text-center
-            - if service.identity_submission(current_identity).present?
-              = link_to "javascript:void(0);", class: 'btn btn-danger delete-submission', data: { id: "#{service.identity_submission(current_identity).first.id}", service_id: "#{service.id}", sr_id: "#{@service_request.id}", protocol_id: "#{@service_request.protocol_id}" } do
-                %span.glyphicon.glyphicon-remove
+    - if service_request.additional_detail_services.present?
+      - service_request.line_items.each do |line_item|
+        - service = Service.find(line_item.service_id)
+        - if service.questionnaires.active.present?
+          %tr
+            %td
+              = service.name
+            %td
+              .text-center
+                - if service.identity_submission(current_identity).present?
+                  = link_to service_additional_details_submission_path(service, service.identity_submission(current_identity).first),
+                    remote: true, class: 'btn btn-primary' do
+                    %span.glyphicon.glyphicon-search
+                - else
+                  = link_to new_service_additional_details_submission_path(service, protocol_id: @service_request.protocol_id, sr_id: @service_request.id, line_item_id: line_item.id), remote: true, class: 'btn btn-primary' do
+                    = t(:additional_details)[:document_complete_details]
+                    %span.glyphicon.glyphicon-modal-window
+            %td
+              .text-center
+                = service.identity_submission(current_identity).present? ? "True" : "False"
+            %td
+              .text-center
+                - if service.identity_submission(current_identity).present?
+                  = link_to edit_service_additional_details_submission_path(service, service.identity_submission(current_identity).first, sr_id: @service_request.id), remote: true, class: 'btn btn-warning' do
+                    %span.glyphicon.glyphicon-edit
+            %td
+              .text-center
+                - if service.identity_submission(current_identity).present?
+                  = link_to "javascript:void(0);", class: 'btn btn-danger delete-submission', data: { id: "#{service.identity_submission(current_identity).first.id}", service_id: "#{service.id}", sr_id: "#{@service_request.id}", protocol_id: "#{@service_request.protocol_id}", line_item_id: "#{line_item.id}" } do
```

Comment:
Apparently, not. Thanks man. Reference - http://stackoverflow.com/questions/31061716/interpolate-variables-into-data-attributes